### PR TITLE
Make compatible with latest frictionless-py

### PR DESCRIPTION
**Pull request by peterdesmet**
See #381

### CODE REVIEW

File: tests/requirements.txt [hunk 1]
```diff
@@ -6,12 +6,12 @@ charset-normalizer==2.0.7
 click==8.0.3
 colorama==0.4.4
 decorator==5.1.0
-frictionless==5.13.1
```
Comment (peterdesmet) [inline tests/requirements.txt:9]: We could update this to 5.17.0 so we always test the latest version (or 5.13.1 to indicate that's where we started on).

**Comment by kbubnicki** [comment]
Looks good, I have also tested this locally. Of course `validate_example_versioned_schema.py` did not pass.

File: tests/requirements.txt (hunk 1 above)
Comment (kbubnicki) [reply tests/requirements.txt:9]: I would test the latest version.

**Review by kbubnicki — APPROVED** [review]

File: tests/requirements.txt (hunk 1 above)
Comment (peterdesmet) [reply tests/requirements.txt:9]: Ok, sounds good to me, so updating to >= 5.17.0 then?

**Pull request merged**